test/controllers/supplementary/ConfirmationPageControllerSpec.scala

```diff
@@ -53,24 +56,47 @@ class ConfirmationPageControllerSpec extends CustomExportsBaseSpec {
       resultAsString must include(messages("supplementary.confirmation.explanation"))
     }
 
-    "display LRN from flash" in new Test {
```

Comment:
are we using flash? shouldn't use the scope on MDTP please use cache

Reply:
It's actually implemented in previous task, not related with actual one. @MaciejRewera check this

Reply:
If we shouldn't, then I will change it to cache in separate PR.